src/app/search/FilterHelp.tsx

```diff
@@ -97,9 +97,10 @@ export default connect<StoreProps>(mapStateToProps)(FilterHelp);
 
 function FilterExplanation({ filter }: { filter: FilterDefinition }) {
   const dispatch = useDispatch();
+  const additionalSuggestions = filter.suggestionsGenerator?.() || [];
   const suggestions = Array.from(
     new Set(
-      generateSuggestionsForFilter(filter).filter(
+      [...generateSuggestionsForFilter(filter), ...additionalSuggestions].filter(
```

Comment:
Does `generateSuggestionsForFilter` not already use `suggestionsGenerator`?

Reply:
No the other way around. Where we have created a `suggestionsGenerator ` we use `generateSuggestionsForFilter ` as the function with tweaked parameters.

Reply:
i had it suggestionsgenerator running alongside the formulaic `format` based one, but i can see nesting them instead. i'll loop back and do that once this is settled.